src/Services/Azure/Kusto/KustoService.cs

```diff
@@ -5,10 +5,6 @@
 using AzureMcp.Commands.Kusto;
 using AzureMcp.Options;
 using AzureMcp.Services.Interfaces;
-using Kusto.Cloud.Platform.Data;
-using Kusto.Data;
-using Kusto.Data.Common;
-using Kusto.Data.Net.Client;
```

Comment:
given the changes in this pr, should we remove the `Microsoft.Azure.Kusto.Data` dependency (from src/AzureMcp.csproj). ?